iib/workers/tasks/opm_operations.py

```diff
@@ -1369,3 +1370,22 @@ def set_opm_version(cls, from_index: Optional[str] = None):
         if index_version in opm_versions_config:
             Opm.opm_version = opm_versions_config.get(index_version)
         log.info("OPM version set to %s", Opm.opm_version)
+
+    @classmethod
+    def get_opm_version(cls):
```

Comment:
so basically this function gives the actual version of the opm like 1.26.4 or 1.40, and Opm.opm_version is directly giving the opm bainry name, like opm-1.26.4

we validate the opm at _validate_multiple_opm_mapping() and at get_binary_versions(), so that should ensure opm-1.26.4 would result in valid opm.
I am thinking if running opm version again could create confusions in the log, and if getting the version value with a string operation on Opm.opm_version and then using Version() on that would be useful?

p.s Opm.opm_version and get_opm_version are sounding same with different values, can we use better names or atleast add a comment to note the difference

Reply:
I was thinking the same, running the `re.search()` on `OPM.opm_version`, however then I saw the value `opm` somewhere in the tests at this class attribute, which made me think running the command would be safer.

Reply:
I just renamed the function to get_opm_version_number

Reply:
I was also checking the mentioned methods, and I do not see, any checks that would ensure that the opm_version would not be something like mentioned `opm`.

Reply:
okay makes sense, one can actually install opm v1.26.4 under filename  opmv1264 and both the validations will pass.